src/pages/vaults/VaultPnlChart.tsx

```diff
@@ -189,11 +189,13 @@ export const VaultPnlChart = ({ className }: VaultPnlChartProps) => {
       <$ChartContainer>
         <$ChartBackground chartBackground={chartDotsBackground} />
         <div tw="flexColumn pl-1 pr-1">
-          {hoveredTime != null && (
-            <div tw="text-color-text-0 font-small-book">
+          <div tw="text-color-text-0 font-small-book">
+            {hoveredTime != null ? (
               <Output value={hoveredTime} type={OutputType.Date} />
-            </div>
-          )}
+            ) : (
+              <>&nbsp;</>
```

Comment:
What happened on the previous iteration when just having `null`?

Reply:
sorry, I forgot to link the ticket: https://linear.app/dydx/issue/TRA-652/nit-update-how-the-date-info-is-presented-in-the-chart

Basically when it's null the equity number pops up the the top of the box then when you hover it gets bumped down and Taehoon wanted it to be consistently positioned.